### DISCUSSION TITLE
Programmatically set options

### DISCUSSION THREAD

**Discussion by pchaganti** [JedWatson/react-select#5612] [Q&A]
I am using two selects. When an option is selected in the first select, I set the value of the options in the second select. This works fine as I set the value of the second select to the first item in the list and it shows the correct selection.

However, when I try to get the value of the displayed option in the select, It returns a null. Is there something that needs to be done to refresh the select when options were modified? 

thanks!

**Comment by manjushsh** [comment]
Hi @pchaganti, can you share a example sandbox?

**Comment by pchaganti** [comment]
Was able to get it to work.

thx!
